sensor_msgs/package.xml

```diff
@@ -15,6 +15,7 @@
   <build_depend>geometry_msgs</build_depend>
   <build_depend>std_msgs</build_depend>
 
+
```

Comment:
nit: seems like a leftover of a previous change

Reply:
yeah, i went through a lot of iterations. I'll squash this out.